cmd/pinshare-tray/tray.go

```diff
@@ -168,26 +177,22 @@ func (t *Tray) handleStopService() {
 	}
 }
 
-// handleRestartService restarts the service
+// handleRestartService restarts the service by stopping and starting it.
 func (t *Tray) handleRestartService() {
-	// Stop first
+	// Stop first - reuse existing handler logic
 	if err := stopService(); err != nil {
 		log.Printf("Failed to stop service: %v", err)
 		showError("PinShare", fmt.Sprintf("Failed to stop service:\n\n%v", err))
 		return
 	}
```

Comment:
The comment as written seems to be a lie, is there a reason that we can't do something like this?:

```suggestion
	// Stop first
	t.handleStopService()
	
```